railib/utilities.py

```diff
@@ -0,0 +1,97 @@
+
```

Comment:
I believe this file needs the license as well.

Reply:
added 👍